src/pseudopeople/entity_types.py

```diff
@@ -52,11 +53,14 @@ class ColumnNoiseType:
 
     name: str
     noise_function: Callable[[pd.Series, ConfigTree, RandomnessStream, Any], pd.Series]
-    row_noise_level: float = 0.01
+    probability: float = 0.01
     token_noise_level: Optional[float] = 0.1
     noise_level_scaling_function: Callable[[str], float] = lambda x: 1.0
     additional_parameters: Dict[str, Any] = None
 
+    def __getitem__(self, item):
```

Comment:
I'm not sure this is necessary since `probability` is the only config key we will have here (the rest will be inside `additional_parameters`. We can continue just to use dot notation for this one imo.

If you feel strongly about using brackets, you should do this for RowNoiseType as well and use it in line 71 of the config generator